src/qt/optionsmodel.h

```diff
@@ -48,6 +49,9 @@ class OptionsModel : public QAbstractListModel
         ThreadsScriptVerif,     // int
         DatabaseCache,          // int
         SpendZeroConfChange,    // bool
+        AddressTypeLegacy,      // bool
+        AddressTypeP2SH,        // bool
+        AddressTypeBIP173,      // bool
```

Comment:
`AddressTypeBech32` would be more consistent with `enum OutputType`.

Reply:
Agree.

Reply:
Fixed.